zmon_worker_monitor/builtins/plugins/sql_postgresql.py

```diff
@@ -129,22 +129,22 @@ def __init__(
             if not m:
                 raise CheckError('Invalid shard connection: {}'.format(shard_def))
             connection_str = ("host='{host}' port='{port}' dbname='{dbname}' user='{user}' password='{password}' "
-                              "connect_timeout='{connect_timeout}' options='-c statement_timeout={timeout}' "
+                              "connect_timeout='{connect_timeout}' "
                               "application_name='ZMON Check {check_id} (created by {created_by})' ").format(
                 host=m.group('host'),
                 port=int(m.group('port') or DEFAULT_PORT),
                 dbname=m.group('dbname'),
                 user=user,
                 password=password,
                 connect_timeout=connect_timeout,
-                timeout=timeout,
```

Comment:
Is it problematic to keep both timeouts, using ``options`` and ``SET``?

Reply:
Not really problematic, but what's the point?  If the first one worked, you don't need the second one.  The 2nd is more reliable, however.